vendor/vendor.json

```diff
@@ -311,5 +317,5 @@
 			"revisionTime": "2018-08-17T08:25:03Z"
 		}
 	],
-	"rootPath": "github.com/buildkite/lifecycled"
+	"rootPath": "github.com/itsdalmo/lifecycled"
```

Comment:
@lox - Before merging, any suggestions about how to fix this? Feels wrong to manually fix it, but I don't know how to fix it using `govendor`?

Reply:
I generally fix this by keeping my forks under the original repo name in my git path. So if it's under `$GOPATH/src/github.com/buildkite/lifecycled` it should work as expected.

I'd be happy to move to Go 1.11 and Go modules if you want!